### DISCUSSION TITLE
Waiting time

### DISCUSSION THREAD

**Discussion by Kirofeed** [community/community#65874] [GitHub Education]
### Hi everyone,

i was applied but still have no access. "We're still processing your application. Once this is complete you will have access to the student developer pack offers." How long should i wait?

**Comment by nirav-gajera** [comment]
The processing time for the GitHub Student Developer Pack application can vary depending on a number of factors, such as the volume of applications that GitHub is receiving at the time. However, in general, you should expect to wait for a few days to a week for your application to be processed.

If it has been more than a week since you applied and you have not yet received access to the Student Developer Pack offers, you can contact [GitHub support](https://support.github.com/) for assistance.
